web/src/components/apps/EmbeddedClusterManagement.tsx

```diff
@@ -479,12 +480,12 @@ const EmbeddedClusterManagement = ({
           )}
           {!rolesData && rolesError && (
             <p className="tw-text-base tw-w-full tw-text-center tw-py-4 tw-text-pink-500 tw-font-semibold">
-              {rolesError?.message}
+              {rolesError?.message || "Unable to fetch roles"}
             </p>
           )}
-          {(rolesData?.roles || NODE_TYPES).length > 1 && (
+          {rolesData?.roles && rolesData?.roles.length > 1 && (
```

Comment:
nit: you don't need the `?` in the second condition because you've already verified it exists in the first condition.

Reply:
ah good catch :)

fixed that in a couple of places in https://github.com/replicatedhq/kots/pull/4437/commits/24cc3e871dbe11cbfeefa1fd5de24e02106fe505